pageserver/src/pgdatadir_mapping.rs

```diff
@@ -729,8 +729,15 @@ impl Timeline {
         let current_policy = self.last_aux_file_policy.load();
         match current_policy {
             Some(AuxFilePolicy::V1) => {
-                warn!("this timeline is using deprecated aux file policy V1 (policy=V1)");
-                self.list_aux_files_v1(lsn, ctx).await
+                let res = self.list_aux_files_v1(lsn, ctx).await?;
+                if !res.is_empty() {
```

Comment:
this should be `let empty_str = if res.is_empty() { ", empty" } else { "" };` plus using that str in formatting. That would be shorter and less repetitive.